src/content/getting-started/observability/logging/disable/index.md

```diff
@@ -0,0 +1,74 @@
+---
+linkTitle: Disable logging
+title: Disable Logging
```

Comment:
This sounds too generic. Sounds like you turn off logging overall, which isn't the case. The article heading needs to make sense standalone i.e. without knowing that it's a child of the "observability"/"logging" topic, so I'd suggest we make this more specific to what customers actually want to do here.

Reply:
We should first rename to something like Disable Managed Logging for my Workload cluster but it should be under the observability topic for sure